docs/_config.yml

```diff
@@ -42,18 +42,25 @@ future: true
 # Exclude non-site files
 exclude: ['bin', 'README.md','Gemfile*','Makefile']
 
-pandoc:
-  skip: true
-  impose: false
-  output: ./tmp
-  flags: '--smart'
-  site_flags: '--toc'
-  outputs:
-    pdf: '--latex-engine=latex'
-    epub:
-    markdown:
+# pandoc:
```

Comment:
Really not a fan of commented-out functionality in files (without a reason). If we don't need it, delete it and we can get it back with version control if necessary